paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1157,7 +1161,7 @@ public static Path path(Options options) {
         return new Path(options.get(PATH));
     }
 
-    public FileFormatType formatType() {
+    public String formatType() {
         return options.get(FILE_FORMAT);
```

Comment:
I think we need to toLowerCase here. Consider:
```
Options options = new Options();
options.setString(CoreOptions.FILE_FORMAT.key(), "AVRO");
System.out.println(options.get(CoreOptions.FILE_FORMAT));
```

Upper case works fine, the change should be compatible to upper case usage.
You can add test in `FileFormatTest`.

Reply:
fixed, and and some tests in `FileFormatTest `.